test/test_samples.rb

```diff
@@ -46,6 +46,9 @@ def test_filename_listed
     Samples.each do |sample|
       if sample[:filename]
         listed_filenames = Language[sample[:language]].filenames
+        if sample[:language] == "Hosts File" # Kludge for https://bit.ly/41EyUkU
+          listed_filenames -= ["HOSTS"]
+        end
```

Comment:
I think we need to consider the INI lanugage here too.

How about:

```suggestion
        listed_filenames -= ["HOSTS"] if ["Hosts File", "INI"].include?(sample[:language])
```